extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQWorkerTaskLauncher.java

```diff
@@ -472,6 +471,32 @@ private void sleep(final long sleepMillis, final boolean shuttingDown) throws In
     }
   }
 
+  /**
+   * Information about current status of running tasks
+   */
+  public static class WorkerStatus
```

Comment:
Let's declare this class in another class file since it's used in controller impl. 
Also, should we just change the name to WorkerCounts and rename the method accordingly?

Reply:
Done